src/Sql/Sql.Test/ScenarioTests/ManagedInstanceCrudScenarioTests.cs

```diff
@@ -33,6 +33,7 @@ protected override void SetupManagementClients(RestTestFramework.MockContext con
 
         public ManagedInstanceCrudScenarioTests(ITestOutputHelper output) : base(output)
         {
+            base.resourceTypesToIgnoreApiVersion = new string[] { "Microsoft.Sql/managedInstances" };
```

Comment:
This will have new recordings - this is stubbed here for tests.

Reply:
In https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/329, I've added some new recordings that verifying server level communication with 2018-06-01-preview against managedInstances works after api version bump.